### PR TITLE
Fix Unit Test

### PR DESCRIPTION
**Pull request by pichaoqkc**
- Fixed contract div 0 bug
- Removed `totalSecondsClaimedX128` related checks
- Fixed all edge cases
- Update gas cost snapshot

### CODE REVIEW

**Review by qizhou — APPROVED** [review]

File: contracts/UniversalV3Staker.sol [hunk 1]
```diff
@@ -436,6 +436,7 @@ contract UniversalV3Staker is IUniversalV3Staker, Multicall {
         uint208 rshareX64 = rshareUpperX64 - rshareLowerX64;
         require(rshareX64 <= rshareUpperX64, 'UniswapV3Staker::calculateReward: sub overflow');
         uint256 rewardX64 = uint256(liquidity) * uint256(rshareX64);
+        if (rewardX64 == 0) return 0;
```
Comment (ninjaahhh) [inline contracts/UniversalV3Staker.sol:439]:
```suggestion
        if (rewardX64 == 0) return 0;

```

**Review by ninjaahhh — APPROVED** [review]
